pages/transactional-email/concepts.mdx

```diff
@@ -16,6 +16,10 @@ categories:
 
 An aggregate report (`rua`) is a periodic summary that provides data about a domain's email authentication activity. It includes information such as the volume of emails received, the percentage of emails that pass or fail DMARC checks, and the sources of failed authentication.
 
+## Blocklist
+
+Blocklist is a feature that allows your to manage email addresses that generate errors, or “bounces”. Emails are automatically blocked when a mailbox does not exist, or is full, to prevent sending emails to a recipient address that is not reachable.
```

Comment:
```suggestion
Blocklist is a feature that allows you to manage email addresses that generate errors, or “bounces”. Emails are automatically blocked when a mailbox does not exist, or is full, to prevent sending emails to a recipient address that is not reachable.
```